packages/web/components/Link.tsx

```diff
@@ -1,13 +1,18 @@
-import { Link } from '@metafam/ds';
+import { Link, LinkProps as DSLinkProps } from '@metafam/ds';
 import NextLink, { LinkProps } from 'next/link';
 import React from 'react';
 
-type Props = LinkProps;
+type MetaLinkProps = LinkProps & DSLinkProps;
 
-export const MetaLink: React.FC<Props> = ({ children, ...props }) => (
-  <NextLink {...props} passHref>
+export const MetaLink: React.FC<MetaLinkProps> = ({
+  children,
+  href,
+  as,
+  ...props
+}) => (
+  <NextLink href={href} as={as} passHref>
```

Comment:
Should make sure we pass in any of the other Next link props to it as well: https://nextjs.org/docs/api-reference/next/link